shared/message/strategy/FallRecovery.proto

```diff
@@ -0,0 +1,27 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2022 NUbots <nubots@nubots.net>
+ */
+syntax = "proto3";
+
+package message.strategy;
+
+/**
+ * Emit a PlanGetUp Task in order to tell the GetUpPlanner to start monitoring the robot's state and get it up if it
+ * falls.
+ */
```

Comment:
doesn't mention relaxing? names are wrong too. maybe don't make this comment reliant on other message names? Something like 'watches the robot's state and relaxes if it is falling over, and gets up when it has fallen' - or even more generic 'watches the robot's state and handles the recovery if the robot is falling/has fallen'.